assemblies/developer-lightspeed-guide/assembly_run-dev-lightspeed-analysis.adoc

```diff
@@ -0,0 +1,36 @@
+:_newdoc-version: 2.18.3
+:_template-generated: 2025-04-08
+
+ifdef::context[:parent-context-of-run-dev-lightspeed-analysis: {context}]
+
+:_mod-docs-content-type: ASSEMBLY
+
+ifndef::context[]
+[id="run-dev-lightspeed-analysis"]
+endif::[]
+ifdef::context[]
+[id="run-dev-lightspeed-analysis_{context}"]
+endif::[]
+= Running an analysis and resolving issues 
+
+:context: run-dev-lightspeed-analysis
+
+After you complete the configurations, the next step is running an analysis to identify the issues in the code and generate suggestions to address the issues. You can run an analysis by using {mta-dl-plugin} in the solution server mode or in the agent mode.
```

Comment:
> You can run an analysis by using {mta-dl-plugin} in the solution server mode or in the agent mode.

This isn't quite right, analysis is the same, it is a rules driven static code analysis.

The solution server is an optional component that a user may decide to enter coordinates for if they want their usage info to be stored for the organization to gain better contextual hints for solutions.

The agentic workflows may be enabled if desired so follow up issues can be attempted to be fixed.